packages/web/lib/hooks/opensea.ts

```diff
@@ -36,11 +36,9 @@ export const useOpenSeaCollectibles = ({
   const [loading, setLoading] = useState<boolean>(false);
   const owner = player.ethereum_address;
   const collectiblesFavorites = useMemo(
-    () =>
-      player && player.box_profile && player.box_profile.collectiblesFavorites
-        ? player.box_profile.collectiblesFavorites
-        : [],
-    [player],
+    // IDX's basic profile doesn't include a list of favorite NFTs
+    () => [], // player?.box_profile?.collectiblesFavorites ?? [],
```

Comment:
```suggestion
```

Reply:
@dysbulic I mean that since `collectibleFavorites` is always `[]`, we can remove the variable and its usage in the below `useEffect` under the first if check.

Reply:
Done.